### PR TITLE
Change rc.interface tone_alarm instances to tune_control.

### PR DESCRIPTION
**Pull request by mcsauder**
Hi,

This PR changes remaining instances of tone_alarm to tune_control in rc.interface.  This commit has been tested on pixracer based hardware although additional testing may be required to cover other hardware configurations.

Please let me know if you have any questions or comments on this PR!

-Mark

### CODE REVIEW

File: ROMFS/px4fmu_common/init.d/rc.interface [hunk 1]
```diff
@@ -181,7 +177,8 @@ then
 	else
 		echo "ERROR  [init] Failed loading mixer: ${MIXER_FILE}"
 		echo "ERROR  [init] Failed loading mixer: ${MIXER_FILE}" >> $LOG_FILE
-		tone_alarm ${TUNE_ERR}
+		# Error tune.
+		tune_control play -t 2
```
Comment (dagar) [inline ROMFS/px4fmu_common/init.d/rc.interface]: How are we supposed to know that `-t 2` is an error?

Comment (mcsauder) [reply ROMFS/px4fmu_common/init.d/rc.interface]: I agree.  I think it would be nice if the tunes were an enum rather than an array.

Comment (mcsauder) [reply ROMFS/px4fmu_common/init.d/rc.interface]: Perhaps default_tunes.cpp and tune_definition.h could be combined, and maybe to simplify things they could be condensed into tunes.cpp and tunes.h?

Comment (dagar) [reply ROMFS/px4fmu_common/init.d/rc.interface]: Can we do that before adding more magic numbers everywhere? @simonegu please advise.

Comment (mcsauder) [reply ROMFS/px4fmu_common/init.d/rc.interface]: @dagar, this looks like an appropriate time to work on the tone_alarm/tune_control refactor.  Updates to follow.

Comment (potaito) [reply ROMFS/px4fmu_common/init.d/rc.interface]: @mcsauder 
> Perhaps default_tunes.cpp and tune_definition.h could be combined, and maybe to simplify things they could be condensed into tunes.cpp and tunes.h?

As far as I know those separate files exist such that manufacturers and individuals can easily create custom tunes without creating an unnecessary diff. It's one of those things "towards px4 modularity". Also I'm not sure what would be made simpler by merging all these files into `tunes.h`.

**Comment by mcsauder** [comment]
Closing in preference of PR #10257.

**Pull request closed without merging**